examples/modular_commands/commandset_basic.py

```diff
@@ -0,0 +1,121 @@
+# coding=utf-8
```

Comment:
Creating an empty __init__.py file in this directory might be a good enough hack to get the examples to run but probably not to work nicely in an IDE

Reply:
Yeah, the layout of the examples doesn't really support this very well.